isomorphe/app.py

```diff
@@ -34,7 +34,9 @@
 
 app = Flask(__name__)
 app.config["SECRET_KEY"] = os.getenv("FLASK_SECRET_KEY", "default-secret-key")
+app.config["TRANSFORM_TIMEOUT"] = -1  # no timeout
```

Comment:
```suggestion
app.config["TRANSFORM_TIMEOUT"] = 3*60*60  # 3 hours 
```

I would rather use a large enough value and avoid potential ghost jobs.